src/Features/Core/Portable/StackTraceExplorer/StackTraceAnalyzer.cs

```diff
@@ -29,16 +31,32 @@ internal static class StackTraceAnalyzer
         public static Task<StackTraceAnalysisResult> AnalyzeAsync(string callstack, CancellationToken cancellationToken)
         {
             var parsedFrames = Parse(callstack, cancellationToken);
-            return Task.FromResult(new StackTraceAnalysisResult(parsedFrames.ToImmutableArray()));
+            return Task.FromResult(new StackTraceAnalysisResult(callstack, parsedFrames.ToImmutableArray()));
         }
 
         private static IEnumerable<ParsedFrame> Parse(string callstack, CancellationToken cancellationToken)
         {
-            foreach (var line in SplitLines(callstack))
+            // if the callstack comes from ActivityLog.xml it has been
+            // encoding to be passed over HTTP. This should only decode 
+            // specific characters like "&gt;" and "&lt;" to their "normal"
+            // equivalents ">" and "<" so we can parse correctly
+            callstack = WebUtility.HtmlDecode(callstack);
```

Comment:
Is this low hanging fruit?

Would be interesting to see what difference the benchmark shows if you comment out this line (since it doesn't look like it's needed for them). If it's a big enough one then might be better to make the parsers smarter and allow for `&gt;` etc

Reply:
It really depends on what all we want to support. The `<` and `>` are examples, but it also does newlines, `$` and probably more. If we need to reduce this copy further then we might find a way to handle this. The good news is that with this implementation it also displays to the user in a more friendly way.

Reply:
Yeah, I know its not trivial, just wondering what impact this has since its called for every stack trace, and presumably does lots of string processing. If it doesn't make a big difference then clearly its not worth the complexity of the alternative, but we should get the data at least.

To be clear, I mean literally comment out this line, and run the benchmark on your machine, and see what the results are, thats all.

Reply:
# Without HtmlDecode

|                     Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
|--------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
| BenchmarkStackTraceParsing | 20.78 ms | 0.104 ms | 0.087 ms | 93.7500 |     - |     - |      2 MB |

# With HtmlDecode 

|                     Method |     Mean |    Error |   StdDev |   Gen 0 |   Gen 1 | Gen 2 | Allocated |
|--------------------------- |---------:|---------:|---------:|--------:|--------:|------:|----------:|
| BenchmarkStackTraceParsing | 20.88 ms | 0.075 ms | 0.062 ms | 93.7500 | 31.2500 |     - |      2 MB |

Reply:
slightly slower, and more Gen 1 allocations, but not a significant change on the benchmark. I could add a larger benchmark to highlight the difference but I think for now it's fine to leave.

Reply:
Yeah, not worth worrying about. I'm pleasantly surprised.